### DISCUSSION TITLE
Game Themes and Inspirations

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/prophets-of-the-labyrinth-Archive#258] [Announcements]
Themes:
1. Contributing to the group is its own reward
1. Reinventing self as argument against "talent"?

Reference Games
- Slay the Spire
- Hades
- Pokemon

**Comment by doubledh** [comment]
difference in type triumphing over difference in scale (as per [other discussion](https://github.com/Imaginary-Horizons-Productions/prophets-of-the-labyrinth/discussions/330))?

**Reply by ntseng** [reply]
Probably not a core theme since we'll be doing the reverse as well in swarm fights.
